### PR TITLE
Gn migrate 60 712

### PR DESCRIPTION
**Pull request by JohannaOtt**
SVC-1470

### CODE REVIEW

File: annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex [hunk 1]
```diff
@@ -198,6 +198,9 @@
     <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_AbgeleiteteHoehenlinieType/1">{http://www.adv-online.de/namespaces/adv/gid/7.1}position</property>
     <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_AbgeleiteteHoehenlinieType/2">{http://www.opengis.net/gml/3.2}MultiCurve</property>
     <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_AbgeleiteteHoehenlinieType/count">2</property>
+    <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_Abschlussflaeche3DType/1">{http://www.adv-online.de/namespaces/adv/gid/7.1}position</property>
+    <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_Abschlussflaeche3DType/2">{http://www.opengis.net/gml/3.2/AbstractGeometry}choice</property>
+    <property name="defaultGeometry:{http://www.adv-online.de/namespaces/adv/gid/7.1}AX_Abschlussflaeche3DType/count">2</property>
```
Comment (florianesser) [inline annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex:203]: Seems unrelated to the change mentioned in the commit message. A lot of `defaultGeometry` changes ended up in this commit that would probably fit better into the "replace GID 6 namespaces and schemas" commit.

File: annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex.alignment.xml [hunk 2]
```diff
@@ -998,15 +998,15 @@ if (point) {
...
-&#xD;
+// alle Eigenschaften mit Geometrien aus Kommunalem Gebiet&#13;
+def geoms = _.geom.findPolygons(_source.links.AX_KommunalesGebiet.p.position.list())&#13;
+&#13;
+// erzeuge ein NamedPlace, wenn ein assoziiertes Kommunales&#13;
+// Gebiet vorhanden ist mit entsprechender Geometrie&#13;
+if (geoms) {&#13;
+	_target{}&#13;
+}&#13;
+&#13;
```
Comment (florianesser) [inline annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex.alignment.xml:1009]: Line-ending changes should go into a separate commit.

File: annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex (hunk 1 above)
Comment (JohannaOtt) [reply annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex]: Done - removed because they are added when adding test data and are not neded for the alignment

File: annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex.alignment.xml (hunk 2 above)
Comment (JohannaOtt) [reply annex-1/mappings7.1/GeographicalNames/aaa-gn-02.halex.alignment.xml:1009]: moved into a separate commit
